frontend/resources/js/helpers/configuration.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import { currencySelector } from './elements';
+
+const getCurrencySelectorCurrencyCode = () => {
+  // the only way I can see to get the active currency selector is by finding the bold item in the dropdown,
+  // then using the data attribute to get the code
+  const option = currencySelector?.querySelector('strong')?.parentElement;
```

Comment:
From what I can tell this doesn't work on ragstock so is probably a bit too brittle. They do have the strong element but they've removed the currency code data attribute. 

The only other place that I can see it in the HTML is here on cornerstone:

```
<main class="body" id="main-content" role="main" data-currency-code="USD">
```

The ragstock theme has the same element albeit pretty different markup, it still has the body class:

```
<div class="body content" data-currency-code="EUR">
```

document.querySelector('.body')?.dataset?.currencyCode

It does seem to be stored in this cookie. It wouldn't be pretty getting it out here either. The only plus is that it's controlled by BC it's perhaps less likely to break the DOM selector.  (Cur: CAD)

![Screenshot 2023-01-16 at 15 31 46](https://user-images.githubusercontent.com/553566/212714925-1300ff33-0707-46a8-b99f-70b11b4675aa.png)

Reply:
Nice, let me investigate.